### PR TITLE
fix for invisible avatars after radius ignore

### PR DESCRIPTION
**Pull request by birarda**
- fixed a bug with interface refusing to render an avatar it believed it was still ignoring

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1146/HighFidelity-Beta-PR10835-9e822de196ed831ee3c35e31eec19f7f74728820.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1146/HighFidelity-Beta-PR10835-9e822de196ed831ee3c35e31eec19f7f74728820.exe

File: libraries/networking/src/NodeList.cpp [hunk 1]
```diff
@@ -239,11 +239,7 @@ void NodeList::reset() {
     LimitedNodeList::reset();
 
     _numNoReplyDomainCheckIns = 0;
-
-    // lock and clear our set of radius ignored IDs
-    _radiusIgnoredSetLock.lockForWrite();
-    _radiusIgnoredNodeIDs.clear();
-    _radiusIgnoredSetLock.unlock();
+;
```
Comment (AndrewMeadows) [inline libraries/networking/src/NodeList.cpp]: What is the purpose if this lonely semicolon?

**Comment by birarda** [comment]
#### Test Plan

This fixes a bug with avatars staying invisible after they enter your bubble and leave. It is caused by packet loss.

Using [clumsy](https://jagt.github.io/clumsy/) add 10-15% packet loss on all UDP flows. You'll want to make sure that there is not so much loss that you disconnect from the domain, since that will complicate testing.

Perform the following test on master and on this PR. On master you may need 10-20 attempts to reproduce the issue.

(1) Run a sandbox and connect to it with 2 Interface clients.
(2) Have 1 client using their space bubble, the other not.
(3) Put the non-bubble client inside the bubble of the other client. Both clients should see the other avatar disappear.
(4) Remove the non-bubble client from the bubble of the other client. Both clients should see the other avatar reappear. On master, with packet loss, you will occasionally see the other avatar who does not have their space bubble on not re-appear after it is no longer inside the bubble.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1153/HighFidelity-Beta-PR10835-be37998b47ad185fe53753e679264d7cc0d6d5c0.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1153/HighFidelity-Beta-PR10835-be37998b47ad185fe53753e679264d7cc0d6d5c0.dmg

**Comment by YuppyPlays** [comment]
Tested:

Was trying to re-produce the issue mentioned within Master, but I'm not getting the error? (Both computers using build 6790, Latest)

Interface A: Clumsy set to Drop 15% - Tested turning bubble on/off repeatedly with user B within bubble zone.
Interface B: Tested turning bubble on/off repeatedly with user A within bubble zone.

The worst case scenario so far is that the avatars sometimes are facing wrong direction for a few seconds whilst bubble is off between turning it on/off. Cannot replicate issue, unless I'm doing something wrong?

**Comment by huffman** [comment]
@YuppyPlays Can you try this instead?

 1. User A
   1. Bubble on
   1. Clumsy set to 15%
 2. User B
   1. Bubble off
   1. Clumsy off, but if this doesn't work you can try turning it on to 15%
 3. Walk B into A's bubble
 4. Walk B out of A's bubble
 5. If A can't see B - good - you've reproduced the issue. If you can, then go back to 3

**Comment by YuppyPlays** [comment]
@huffman 
No problem, will test shortly- just to confirm, should I be testing on the previous master version 6790 or use the latest version 6807?

**Comment by huffman** [comment]
@YuppyPlays You can use either one

**Comment by YuppyPlays** [comment]
@huffman  Hi Huffman, so I tested this on 2 computers with the following results happening.

User A - Bubble on **Check**
Clumsy set to 15% **Check**
User B - Bubble off **Check**
Clumsy off, but if this doesn't work you can try turning it on to 15% **Check, after initial seemed normal tried with 15% on both computers**
Walk B into A's bubble **Check**
Walk B out of A's bubble **Check**
If A can't see B - good - you've reproduced the issue. If you can, then go back to 3 **I cannot see B until they leave the bubble area, but I thought this was normal behaviour? they always re-appear after exiting the bubble area. Tried with clumsy on both computers, this is over a LAN network, is this possibly causing the issue to not repro?**

**Comment by birarda** [comment]
@YuppyPlays I'll follow up with you to make sure that the settings in clumsy are right. It should have no problem inducing loss over a LAN too.

**Comment by YuppyPlays** [comment]
Testing.

**Comment by YuppyPlays** [comment]
I've managed to repro the  issue within master build thanks to @birarda 's help :)

I'm just having a small issue downloading the build, anyone else able to click it successfully? I'm getting the following:
image

**Comment by zfox23** [comment]
@YuppyPlays that means Gustavo either failed to push the build or deleted it for some reason. Rebuilding is necessary.

**Comment by zfox23** [comment]
test this please

**Comment by YuppyPlays** [comment]
@zfox23 ah I see, will wait for the rebuild :D Thank you

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1464/HighFidelity-Beta-PR10835-be37998b47ad185fe53753e679264d7cc0d6d5c0.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1464/HighFidelity-Beta-PR10835-be37998b47ad185fe53753e679264d7cc0d6d5c0.exe

**Comment by YuppyPlays** [comment]
Resuming testing.

**Comment by YuppyPlays** [comment]
Testing complete:

Repro in Master: Within about 20 times going back and fourth, the issue occurred twice.
Test in PR:  The error is not present **Pass**, However, I did notice once user A did not disappear when user B walked into. But this was with Clumsy on, other than that everything worked as expected.

Without clumsy, also all good.

**Pull request merged**